docs/ReleaseNotes.md

```diff
@@ -6,10 +6,11 @@ Build 042
 Published as version 2.23.2
 
 Bug fixes:
-* Removed unused dependencies
-* Updated remaining dependencies
-* Converted all unit tests from nodeunit to jest
-* Removed the npm-shrinkwrap.json in repository
+* Removed unused dependencies.
+* Updated remaining dependencies.
+* Converted all unit tests from nodeunit to jest.
+* Removed the `npm-shrinkwrap.json` in repository.
+* Added a new `getProjectType()` method on the Project class.
```

Comment:
This is a new feature, so it should cause the version to go to v2.24.0

Reply:
I updated it